.azure/pipelines/components-e2e-tests.yml

```diff
@@ -74,6 +74,10 @@ jobs:
       condition: always()
 
     artifacts:
+    - name: Components_E2E_Logs
+      path: artifacts/log/
+      publishOnError: true
+      includeForks: true
```

Comment:
💟

Reply:
😆 The tests don't actually produce any logs, at least not through the normal channels our other tests do. Something the blazor team should probably remedy.